README.MD

```diff
@@ -72,24 +71,63 @@ This command should print the installed `olm` version.
 
 Note: The provided instructions assume a certain level of system familiarity. Adjustments might be necessary based on your specific setup or if you encounter any issues during the installation process.
 
+## Running a single bot
+
 ### Build
 
 ```bash
-go build -o bot cmd/camino-messenger-bot/main.go
+./scripts/build.sh
 ```
 
 ### Create Configuration
 
-Create a configuration YAML file based on the provided example "camino-messenger-bot.yaml.example".
+Create a configuration YAML file (e.g. `camino-messenger-bot.yaml`) based on the provided example `camino-messenger-bot-supplier.yaml.example` or `camino-messenger-bot-distributor.yaml.example`.
+
+
 
 ### Start the Application
 
 ```bash
-./bot --config camino-messenger-bot.yaml
+./build/bot --config camino-messenger-bot.yaml
 ```
 
 Feel free to explore and customize the application based on your specific requirements. If you encounter any issues or have questions, please refer to the documentation or open an issue in the repository.
 
+## Running both Supplier and Distributor at the same time
+For development proposes, you may want to run both Supplier and Distributor bot at the same time.  
+### Build 
+```bash
+./scripts/build_both.sh
+```
+to build bots to different folders. 
+
+### Create Configuration
+
+Use `camino-messenger-bot-distributor.yaml.example` and `camino-messenger-bot-supplier.yaml.example` files to create configuration YAML files (e.g. `camino-messenger-bot-distributor.yaml` and `camino-messenger-bot-supplier.yaml`). 
+
+### Start the Application
+
+To run the bots, in 2 separate terminals run:  
+```bash 
+./build/distributor/bot --config camino-messenger-bot-distributor.yaml
+```  
+
+```bash 
+./build/supplier/bot --config camino-messenger-bot-supplier.yaml
+```
+
+Or configure the launching file of your preferred IDE to do the same.
+
+⚠️ WARNING:  
+ YAML **configuration files** contain private keys and thus **should never be committed**. We suggest adding any configuration file names to [.git/info/exclude](`camino-messenger-bot-distributor.yaml.example`) to have git ignore them without modifying the .gitignore.
+
```

Comment:
Why do we need this warning? We already put the config files in the `.gitignore` file. 

Also, the `*.yaml.example` files are not config files and they do not contain private keys, the `*.yaml` files do.

Reply:
Yeah, I think warning is a bit excessive. Its more or less common sense to not commit your private data to repos :P